Microsoft.Toolkit.Graph.Controls/Providers/MsalConfig.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Graph.Providers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class MsalConfig : IGraphConfig
```

Comment:
Were we going to move the Msal specific stuff to its own package to be able to isolate MSAL as a dependency? Or @nmetulev did we determine the next Graph version makes that a moot point?

Reply:
I think the packages should be chopped up by provider. That'll make it easier to take a dependency on the one you need, and avoid the unnecessary bits. That's how MGT does it :)

Reply:
I chopped them up. Let me know what you think